src/collective/nitf/tests/test_adapters.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from collective.nitf.config import PLONE_VERSION
+from collective.nitf.testing import INTEGRATION_TESTING
+from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
+
+if PLONE_VERSION >= '4.3':
```

Comment:
``try``/``except`` also here; we need to install collective.syndication when in PLone 4.2